microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/MultipleJarDeploymentTest.java

```diff
@@ -0,0 +1,96 @@
+ */
+@RunWith(Arquillian.class)
+@ServerSetup(FaultToleranceServerSetup.class) // Enables/Disables fault tolerance extension/subsystem for Arquillian in-container tests
+public class MultipleJarDeploymentTest {
+
+    @Inject
+    HelloService helloService;
+
+    private static final String FIRST_DEPLOYMENT_JAR = "first-deployment";
+    private static final String SECOND_DEPLOYMENT_JAR = "second-deployment-jar";
+
+    @Deployment(name = FIRST_DEPLOYMENT_JAR, order = 1, testable = false)
+    public static Archive<?> createFirstJarDeployment() {
+        String mpConfig = "hystrix.command.default.execution.timeout.enabled=true";
+
+        return ShrinkWrap.create(JavaArchive.class, FIRST_DEPLOYMENT_JAR + ".jar")
+                .addClasses(HelloService.class, MyContext.class, HelloFallback.class, FallbackHandler.class)
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
+                .addAsManifestResource(new StringAsset(mpConfig), "microprofile-config.properties");
+    }
+
+    @Deployment(name = SECOND_DEPLOYMENT_JAR, order = 2)
+    public static Archive<?> createSecondJarDeployment() {
+        String mpConfig = "hystrix.command.default.execution.timeout.enabled=false";
+
+        return ShrinkWrap.create(JavaArchive.class, SECOND_DEPLOYMENT_JAR + ".jar")
+                .addClasses(HelloService.class, MyContext.class, HelloFallback.class, FallbackHandler.class)
+                .addClasses(MultipleJarDeploymentTest.class, LogChecker.class)
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
+                .addAsManifestResource(new StringAsset(mpConfig), "microprofile-config.properties");
+    }
+
+    /**
+     * @tpTestDetails Deploy first and then second MP FT application. Both of them are the same (same classes/methods)
+     * @tpPassCrit Verify that second deployment does NOT change MP FT/Hystrix configuration.
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @RunAsClient
+    public void testDeploySecondJarLogsHystrixNoConfigChangeWarning() throws Exception {
+        try (final OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            final LogChecker logChecker = new ModelNodeLogChecker(client, 10, true);
+
+            Assert.assertTrue(
+                    "Log does not contain WARNING Log: WFLYMPFTEXT0002: Hystrix was already configured! Skipping configuration from deployment - but it should.",
+                    logChecker.logMatches(Pattern.compile(".*WFLYMPFTEXT0002.*")));
+            Assert.assertFalse("Log must not contain any ERROR logs but there are. See server log." + logChecker.toString(),
+                    logChecker.logMatches(Pattern.compile(".*ERROR.*")));
+        }
+    }
+
+    /**
+     * @tpTestDetails Deploy first and then second MP FT application. Verify they're working.
+     * @tpPassCrit Verify that second deployment does NOT change MP FT/Hystrix configuration.
```

Comment:
Looking at Javadoc, there's almost no difference with respect to previous method. How about adding a sentence which would explain the actual criteria? Just take inspiration from the test method name, I'd say, but just my 2 cents.

Reply:
fixed